ui/app/templates/components/tool-actions-form.hbs

```diff
@@ -1,3 +1,59 @@
 <form onsubmit={{action "doSubmit"}}>
-  {{partial (concat "partials/tools/" selectedAction)}}
+  {{#if (eq selectedAction 'hash')}}
+    <ToolHash
+      @onClear={{action "onClear"}}
+      @sum={{sum}}
+      @algorithm={{algorithm}}
+      @errors={{errors}}
+      @format={{format}}
+      @input={{input}}
+    />
+  {{else if (eq selectedAction 'random')}}
+    <ToolRandom
+      @onClear={{action "onClear"}}
+      @random_bytes={{random_bytes}}
+      @errors={{errors}}
+      @format={{format}}
+      @bytes={{bytes}}
+    />
+  {{else if (eq selectedAction 'rewrap')}}
+    <ToolRewrap
+      @onClear={{action "onClear"}}
+      @rewrap_token={{rewrap_token}}
+      @selectedAction={{selectedAction}}
+      @errors={{errors}}
+      @token={{token}}
+      @bytes={{bytes}}
+    />
+  {{else if (eq selectedAction 'unwrap')}}
+    <ToolUnwrap
+      @onClear={{action "onClear"}}
+      @unwrap_data={{unwrap_data}}
+      @unwrapActiveTab={{unwrapActiveTab}}
+      @details={{details}}
+      @errors={{errors}}
+      @token={{token}}
+    />
+  {{else if (eq selectedAction 'lookup')}}
+    <ToolLookup
+      @creation_time={{creation_time}}
+      @creation_ttl={{creation_ttl}}
+      @creation_path={{creation_path}}
+      @expirationDate={{expirationDate}}
+      @selectedAction={{selectedAction}}
+      @token={{token}}
+      @onClear={{action "onClear"}}
+      @errors={{errors}}
+    />
+  {{else if (eq selectedAction 'wrap')}}
```

Comment:
notice these are all else ifs, there is no fallback component.  Technically there doesn't need to be, but curious if folks take issue with this.

Reply:
I think this is the right call, unless we want to add an empty state (which would most likely be viewed by a developer updating this space)

Reply:
Good idea with the empty state.  It feels strange to have no else so I'll go ahead and add it.